tests/api2/test_273_kubernetes.py

```diff
@@ -0,0 +1,59 @@
+import os
+import pytest
+import sys
+# from pytest_dependency import depends
+apifolder = os.getcwd()
+sys.path.append(apifolder)
+from functions import GET, PUT, wait_on_job
+from auto_config import ha, scale, pool_name, interface, ip, dev_test
+
+if dev_test:
+    reason = 'Skip for testing'
+else:
+    reason = 'Skipping test for HA' if ha else 'Skipping test for SCALE'
+# comment pytestmark for development testing with --dev-test
+pytestmark = pytest.mark.skipif(ha or not scale or dev_test, reason=reason)
+
+
+def test_01_get_kubernetes_bindip_choices():
+    results = GET('/kubernetes/bindip_choices/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    assert results.json()['0.0.0.0'], results.text
+
+
+def test_02_setup_kubernetes(request):
+    global payload
+    gateway = GET("/network/general/summary/").json()['default_routes'][0]
+    payload = {
+        'pool': pool_name,
+        'route_v4_interface': interface,
+        'route_v4_gateway': gateway,
+        'node_ip': '0.0.0.0'
+    }
+    results = PUT('/kubernetes/', payload)
+    assert results.status_code == 200, results.text
+    job_id = results.json()
+    job_status = wait_on_job(job_id, 180)
```

Comment:
This can take quite some as k8s would be initialized from scratch - let's please change the time to 5 minutes as this is just going to be a VM where this would be running i think.

Reply:
I can update that.